benchmarks/__init__.py

```diff
@@ -0,0 +1,4 @@
+import pyccl as ccl
+
+# Enable caching for testing.
+ccl.Caching.enable()
```

Comment:
Why? I'd rather do the tests without caching to avoid potential bugs.

Reply:
Addressed above.